Source/Scene/ImplicitSubtree.js

```diff
@@ -331,6 +331,8 @@ function initialize(subtree, json, subtreeView, implicitTileset) {
   if (hasExtension(subtreeJson, "3DTILES_multiple_contents")) {
     subtreeJson.contentAvailabilityHeaders =
       subtreeJson.extensions["3DTILES_multiple_contents"].contentAvailability;
+  } else if (Array.isArray(subtreeJson.contentAvailability)) {
```

Comment:
I would add comments explaining which parts are the current schema and which ones are from the older schema.